android/assets/jsons/Civ V - Gods & Kings/TileResources.json

```diff
@@ -501,7 +483,7 @@
 		"gold": 1,
 		"food": 1,
 		"improvement": "Mine",
-		"improvementStats": {"food": 1,"production":1},
+		"improvementStats": {"food": 1},
```

Comment:
Wikia says the old stats?

https://civilization.fandom.com/wiki/Salt_(Civ5)

Reply:
+1 production is provided by mine.